liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -322,13 +323,29 @@ private String getVMLocalConnectorAddress(VirtualMachine wlpvm)
          throws IOException {
       String serviceURL;
       String PROPERTY_NAME = "com.sun.management.jmxremote.localConnectorAddress";
-
+      
       serviceURL = wlpvm.getAgentProperties().getProperty(PROPERTY_NAME);
 
       // On some environments like the IBM JVM the localConnectorAddress is not
       // in the AgentProperties but in the SystemProperties.
       if (serviceURL == null)
          serviceURL = wlpvm.getSystemProperties().getProperty(PROPERTY_NAME);
+      
+      if (serviceURL == null) {
+          File jmxAddr = new File(getServerOutputDir() + "/workarea/com.ibm.ws.jmx.local.address");
```

Comment:
you shouldn't be reading this file. Instead you should be looking at the equivalent file in the logs/state folder.